adafruit_rgb_display/rgb.py

```diff
@@ -293,13 +333,13 @@ def write(self, command=None, data=None):
             with self.spi_device as spi:
                 spi.write(data)
 
-    def read(self, command=None, count=0):
+    def read(self, command: Optional[int] = None, count: int = 0) -> ByteString:
         """SPI read from device with optional command"""
         data = bytearray(count)
         self.dc_pin.value = 0
         with self.spi_device as spi:
             if command is not None:
-                spi.write(bytearray([command]))
+                spi.write(bytearray([command]))  # change to self.write()
```

Comment:
whats up with the comment about changing to self here? Maybe that is part to do with the added functions I asked about in a seperate comment?

Reply:
i noticed it in the code, ill remove